### DISCUSSION TITLE
[Suggestion] Wood and stone texture paints, like the metal ones

### DISCUSSION THREAD

**Discussion by ultrak2k** [tsunamayo/Starship-EVO#4359] [Suggestions]
Hello, I got back into the game after a year (got a new GPU that can run it), and have started building again, however I mostly either build buildings or buildings as bridges for ships, and I noticed that a lot of new paint options exist like custom paints, so I'm curious, could we get a wood and stone texture paint? Perhaps even different colours? I noticed the metallic paints so I was curious of this was possible, it would be excellent for building buildings, especially with the new 0.125m blocks.

**Comment by tsunamayo** [comment]
@ultrak2k Hi, welcome back. Yes I will see what I can do!
